src/xml2js.coffee

```diff
@@ -402,6 +402,23 @@ class exports.Parser extends events.EventEmitter
       else if @saxParser.ended
         throw err
 
+  parseStringSync: (str) =>
+    #assumption 1: sax events block when given a string instead of a stream
+    retval = undefined
+    err = undefined
+    cb = (_err, _retval) ->
+      retval = _retval
+      err = _err
+      return
```

Comment:
This return is not required, since the return value won't be used anywhere, so might as well implicitly `return err = _err`.